tests/run/builtin_divmod.py

```diff
@@ -1,68 +1,236 @@
 # mode: run
```

Comment:
I haven't yet looked in detail at the rest of the PR, but while you're doing this it might be possible to remove this file from `limited_api_bugs.txt`. I think it was only the include for the C `div` function that's missing, so if you've removed that it'll probably just work.